src/png.imageio/pngoutput.cpp

```diff
@@ -24,6 +24,9 @@ class PNGOutput final : public ImageOutput {
         return (feature == "alpha" || feature == "ioproxy"
 #ifdef PNG_eXIf_SUPPORTED
```

Comment:
Just out of curiosity, do you know which version of libpng first had PNG_eXIf_SUPPORTED? Is it older than our minimum version? Or do we span the beginning of exif support?